src/meta/src/manager/catalog/mod.rs

```diff
@@ -2356,7 +2363,8 @@ impl CatalogManager {
         // TODO: Here we reuse the `creation` tracker for `alter` procedure, as an `alter` must
         // occur after it's created. We may need to add a new tracker for `alter` procedure.
         if database_core.has_in_progress_creation(&key) {
-            bail!("table is in altering procedure");
+            // bail!("table is in altering procedure");
```

Comment:
Did you meet any failure here while doing any local test? Better to resolve it.

Reply:
Oh, we need to check the table version as well for create sink into table, since you need to replace the whole table streaming job as well like `alter table`.